doc/tutorials/crypto.md

```diff
@@ -0,0 +1,106 @@
+# Cryptographic Methods in Elektra
+
+Elektra can protect the following aspects of your configuration:
+
+1. confidentiality, and
+2. integrity.
+
+Elektra provides two plugins to achieve this protection:
+
+1. `crypto`, and
+2. `fcrypt`.
+
+## Configuration File Encryption/Decryption
+
+The `fcrypt` plugin enables the encryption and decryption of entire configuration files, thus protecting the confidentiality of the configuration values.
+`fcrypt` utilizes GnuPG (GPG) for all cryptographic operations.
+The GPG key, which is used for encryption and decryption, is specified in the backend configuration under `encrypt/key`.
+You MUST be in possesion of the private key, otherwise `fcrypt` will deny all operations.
+
+Let's assume your GPG private key has the ID `DDEBEF9EE2DC931701338212DAF635B17F230E8D`.
+If you want to encrypt your configuration file `test.ini` you can mount the backend under `user/test` like this:
+
+	kdb mount test.ini user/test fcrypt "encrypt/key=DDEBEF9EE2DC931701338212DAF635B17F230E8D" ini
+
+As a result the file `test.ini` is encrypted using GnuPG.
```

Comment:
At the moment the file was not even written, it might not even exist (no kdb set up to now). And even if it exist, what would happen with the plain text file present there before?